### PR TITLE
[dev-env] Primary domain prompt for primary domain redirect

### PR DESCRIPTION
**Pull request by pschoffer**
## Description

When using the `@` notation we should prompt for domain. Eg:

```
$ vip dev-env @2891 create 
This is a wizard to help you set up your local dev environment.

Sensible default values were pre-selected for convenience. You may also choose to create multiple environments with different settings using the --slug option.


✔ WordPress site title · pschoffer-develop.go-vip.net
✔ Multisite (pschoffer-develop.go-vip.net is NOT multisite) (y/N) · false
✔ Would you like to redirect to pschoffer-develop.go-vip.net for missing media files? (Y/n) · true

```

### CODE REVIEW

File: src/bin/vip-dev-env-create.js [hunk 1]
```diff
@@ -84,6 +84,7 @@ command()
 			debug( `WARNING: ${ message }`, error.message );
 			console.log( chalk.yellow( 'Warning:' ), message );
 		}
+		debug( 'App Info', appInfo );
```
Comment (pauarge) [inline src/bin/vip-dev-env-create.js]: Is this a development leftover?

**Review by pauarge — CHANGES_REQUESTED** [review]
Small question, but looking good

File: src/bin/vip-dev-env-create.js (hunk 1 above)
Comment (pschoffer) [reply src/bin/vip-dev-env-create.js]: Yes:)

Comment (pschoffer) [reply src/bin/vip-dev-env-create.js]: Removed

**Review by pauarge — APPROVED** [review]
Looking good

**Pull request merged**